packages/client/components/TeamHealthVotingRow.tsx

```diff
@@ -0,0 +1,54 @@
+import styled from '@emotion/styled'
+import graphql from 'babel-plugin-relay/macro'
+import React from 'react'
+import {useFragment} from 'react-relay'
+import {PALETTE} from '../styles/paletteV3'
+import {PokerCards} from '../types/constEnums'
+import {TeamHealthVotingRow_stage$key} from '../__generated__/TeamHealthVotingRow_stage.graphql'
+import AvatarList from './AvatarList'
+import {Check as CheckIcon} from '@mui/icons-material'
+import MiniPokerCard from './MiniPokerCard'
+import PokerVotingNoVotes from './PokerVotingNoVotes'
+
+interface Props {
+  stage: TeamHealthVotingRow_stage$key
+  isInitialStageRender?: boolean
+}
+
+const MiniCardWrapper = styled('div')({
+  // This adds the gutter between the mini card and the avatars
+  marginRight: 16
+})
+
+const TeamHealthVotingRow = (props: Props) => {
+  const {stage: stageRef, isInitialStageRender} = props
+  const stage = useFragment(
+    graphql`
+      fragment TeamHealthVotingRow_stage on TeamHealthStage {
+        votedUsers {
+          ...AvatarList_users
+        }
+      }
+    `,
+    stageRef
+  )
+  const {votedUsers} = stage
+  return (
+    <div className='flex h-14 w-80 items-center rounded bg-slate-300 py-[5px] pl-4'>
+      <MiniCardWrapper>
+        <MiniPokerCard color={PALETTE.JADE_400}>
+          <CheckIcon className='text-white' />
+        </MiniPokerCard>
+      </MiniCardWrapper>
+      <AvatarList
+        size={PokerCards.AVATAR_WIDTH}
+        users={votedUsers}
+        isAnimated={!isInitialStageRender}
+        borderColor={PALETTE.SLATE_300}
+        emptyEl={<PokerVotingNoVotes />}
```

Comment:
+1 to me, the green checkmark indicates that something has been successful, even though no votes have happened yet. I find that slightly misleading, but it is a pattern that exists in Poker. I'd rather just say "No Votes" without the green card, or perhaps the card could have a no entry emoji instead of a tick

![no-votes](https://github.com/ParabolInc/parabol/assets/39854876/166b16e3-4b9f-4a27-9c57-dcb3033aff76)

Reply:
I've updated it to be closer to the original design, the check is now only full green once the user voted.